components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -185,7 +185,7 @@ extension Nimbus: FeaturesInterface {
             Glean.shared.setExperimentActive(
                 experiment.slug,
                 branch: experiment.branchSlug,
-                extra: ["enrollmentId": experiment.enrollmentId]
+                extra: nil
```

Comment:
Do we need `extra` here at all? Can we just remove it?

Reply:
Hmm, strange.

The Glean method signature in swift is:

```swift
public func setExperimentActive(_ experimentId: String, branch: String, extra: [String : String]?)
```